src/sentry/api/endpoints/sentry_app_publish_request.py

```diff
@@ -16,19 +16,19 @@ def post(self, request, sentry_app):
         if sentry_app.is_internal:
             return Response({"detail": "Cannot publish internal integration"}, status=400)
 
-        # For now, just send an email that a request to publish has been amde
-        message = "User %s of organization %s wants to publish %s" % (
+        message = "User %s of organization %s wants to publish %s\n" % (
             request.user.email,
             sentry_app.owner.slug,
             sentry_app.slug,
         )
 
+        for question_pair in request.data.get("questionnaire"):
+            message += "\n\n>%s\n%s" % (question_pair["question"], question_pair["answer"])
+
+        subject = "Sentry Integration Publication Request from %s" % sentry_app.owner.slug
+
         email.send_mail(
-            "Sentry App Publication Request",
-            message,
-            options.get("mail.from"),
-            ["partners@sentry.io"],
-            fail_silently=False,
+            subject, message, options.get("mail.from"), ["partners@sentry.io"], fail_silently=False
```

Comment:
How will `Sentry Apps` in general work for OSS?

Reply:
@billyvg What's OSS?

Reply:
open source [software]

Reply:
I don't know what the right answer is, maybe it's as simple as disabling `Publish` for OSS installs.